content/publishing/email-and-slack-notifications.md

```diff
@@ -0,0 +1,33 @@
+---
+description: Receive artifacts of successful builds and build status updates with email or Slack.
+title: Email and Slack notifications
+weight: 8
+---
+
+## Email
+
+Email publishing settings can be found in **App settings > Publish > Email**.
+
+Email publishing is the only publishing option which is set enabled (`Enable email publishing`) by default. Default email from the service used to log in (Github, Bitbucket, Gitlab) is used. You can fill in multiple emails.
```

Comment:
_Default email from the service used to log in (Github, Bitbucket, Gitlab) is used._  Needs rephrasing

Reply:
tried to rephrase in d11b88d1f35a70c0e25786f0d8401944b9476f2f